index.html

```diff
@@ -30,16 +30,43 @@ <h1 style="color: #49a1e0"> <img src="static/shabdawali_logo.png" height="50px"
                 <div class="row" style="height: 45%">
                     <div class="col-12 d-flex justify-content-center align-items-end" >
                         <div>
-                            <!-- display blinking cursor checkbox. -->
                             <p>
-                                <input type="checkbox" id="check-blinkCursor" name="check-blinkCursor" checked>
-                                <label for="check-blinkCursor">Display Blinking Cursor.</label>
-
-                                <input type="checkbox" id="check-keySound" name="check-keySound">
-                                <label for="check-keySound">Play keypress sound.</label>
-
-                                <input type="checkbox" id="check-dynamicPause" name="check-dynamicPause" checked>
-                                <label for="check-dynamicPause">Dynamic pauses</label>
+                                <table width="100%"> <!--Added Table to aligning the checkboxes and button-->
```

Comment:
As we're using the Bootstrap CSS framework, you can achieve it without the table tag but `<div class="row">..`

Reply:
@ashishpatelcs  suggested to leave the checkboxes as is. I used the table tag only to align the newly added button 'Edit Text'. If this is part of a separate PR then i will remove the table tags completely.

Reply:
@asingh217 Yes you can remove the table tags. The buttons can be aligned using a div, if necessary.